src/app/[locale]/(pages)/profileRestore/page.tsx

```diff
@@ -1,42 +1,52 @@
 'use client';
 import { Button, Grid, Typography } from '@mui/material';
+import { useTranslations } from 'next-intl';
 import Link from 'next/link';
 import { FAQ } from '../../_component/accordion/faqDefault';
 import { BackButton } from '../../_component/backButton/backButton';
 import { IdpListOnApp } from '../../_component/idpListOnApp/idpListOnApp';
 import { Introduction } from '../../_component/introduction/introduction';
+import { Flows } from '../../_enums/Flows';
 import { commonBackgroundWithBack } from '../../_utils/styles';
 import { ROUTES } from '../../_utils/routes';
 
-const RestoreProfile = (): React.ReactElement => (
-  <>
-    <Grid sx={commonBackgroundWithBack} xs={12} sm={12}>
-      <BackButton />
-      <Introduction
-        title={"Vuoi ripristinare l'accesso a IO?"}
-        summary={
-          "Se hai messo in sicurezza la tua identità digitale puoi ripristinare l'accesso a IO e entrare nuovamente in app con tutte le tue identità."
-        }
-        summaryColumns={{ xs: 12, md: 7.5 }}
-      />
+const RestoreProfile = (): React.ReactElement => {
+  const t = useTranslations('ioesco');
+  return (
+    <>
+      <Grid sx={commonBackgroundWithBack} xs={12} sm={12}>
+        <BackButton />
+        <Introduction
+          title={t('restore.restoreaccess')}
+          summary={t('restore.login-1')}
+          summaryColumns={{ xs: 12, md: 7.5 }}
+        />
 
-      <Grid container flexDirection={'column'}>
-        <IdpListOnApp />
-        <Grid item sm={10} md={7}>
-          <Typography mb={5} fontSize={'20px'}>
-            Per ripristinare l&apos;accesso all&apos;app IO ti serve il <b>codice sblocco</b>
-            fornito in fase di blocco.
-          </Typography>
+        <Grid container flexDirection={'column'}>
+          <IdpListOnApp />
+          <Grid item sm={10} md={7}>
+            {/* TODO: [TL] tags */}
+            <Typography mb={5} fontSize={'20px'}>
+              {t('restore.insertcode')}
+              {/* Per ripristinare l&apos;accesso all&apos;app IO ti serve il <b>codice sblocco</b>
```

Comment:
why there is this comment?

Reply:
I left the previous paragraphs commented as a memo to compare where and which are the tags we have to put in the text. Every one of these has a TODO to fix it when we'll have the correct texts with tags. 

Can you give me a new task so I can change the TODOs into FIXMEs with the new task number in order to pass this PR and come back later on these?